pkg/scene/list.go

```diff
@@ -0,0 +1,69 @@
+package scene
+
+import "github.com/reearth/reearth-backend/pkg/id"
+
+type List struct {
```

Comment:
```suggestion
type ClusterList struct {
```

Too abstract name for "scene" package. This type is used as following:

```go
import "github.com/reearth/reearth-backend/pkg/scene"

func hoge() {
	clusters := scene.List{} // hm, it's a list of the scene?
	clusters := scene.ClusterList{} // ok, it's a list of the clusters.
}
```